core/primitives/src/epoch_manager.rs

```diff
@@ -139,6 +149,20 @@ impl AllEpochConfig {
             config.chunk_producer_kickout_threshold = 80;
             config.validator_selection_config.num_chunk_only_producer_seats = 200;
         }
+
+        // Adjust the number of block and chunk producers for all chains except
+        // mainnet, to make it easier to test the change.
+        if chain_id != "mainnet"
+            && checked_feature!("stable", TestnetFewerBlockProducers, protocol_version)
```

Comment:
This will include mocknet.

Reply:
Yes, and that is intentional. I'd like to test this change in mocknet.

Reply:
Hmm, okay but I am a bit weary about diverging mocknet and mainnet.